test/commands/append.js

```diff
@@ -48,6 +48,13 @@ describe("the append command", function () {
 		div.innerHTML.should.equal("Content");
 	});
 
+	it("can append a value to I", function () {
+		var div = make(`<div _="on click 
+                            append 'Content' to I"></div>`);
```

Comment:
wouldn't this be `append content to me` or do you want to test `I` specifically?

Reply:
`append content to me` works, while `append content to I` fails.